### PR TITLE
Ger b event overall header headerbar append

### PR DESCRIPTION
**Pull request by GerB**
[Template event] overall_header_headerbar_append

Add event for appending content to headerbar (such as extra menu)

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [ ] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [ ] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

https://tracker.phpbb.com/browse/PHPBB3-14785

### CODE REVIEW

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -1375,6 +1375,13 @@ overall_header_head_append
 * Since: 3.1.0-a1
 * Purpose: Add asset calls directly before the `</head>` tag
 
+overall_header_headerbar_append
+===
+* Locations:
+    + styles/prosilver/template/overall_header.html
+* Since: 3.1.9
```
Comment (Crizz0) [inline phpBB/docs/events.md]: 3.1.9 is already released. Please change this to 3.1.10-RC1

File: phpBB/styles/prosilver/template/overall_header.html [hunk 2]
```diff
@@ -100,6 +100,7 @@
 			<!-- ENDIF -->
 
 			</div>
+			<!-- EVENT overall_header_headerbar_append -->
```
Comment (Crizz0) [inline phpBB/styles/prosilver/template/overall_header.html]: Append is for inline Codes, after for a new line.

**Comment by Crizz0** [comment]
When there is an after, does a before make sense too?

**Comment by GerB** [comment]
I don't know if every after needs a before?

**Comment by GerB** [comment]
Accidentally closed

**Comment by Crizz0** [comment]
I think a "before"-template-event is useful here. Just put in front of the headerbar, like you have put the "after" right after it.

File: phpBB/docs/events.md [hunk 3]
```diff
@@ -1375,6 +1375,20 @@ overall_header_head_append
...
 * Purpose: Add asset calls directly before the `</head>` tag
 
+overall_header_headerbar_before
+===
+* Locations:
+    + styles/prosilver/template/overall_header.html
+* Since: 3.1.10-RC1
+* Purpose: Add content at the beginning of the headerbar
+* 
+overall_header_headerbar_after
```
Comment (marc1706) [inline phpBB/docs/events.md]: Events are sorted alphabetically which means the _after event needs to be before the _before event in this document.

File: phpBB/docs/events.md [hunk 4]
```diff
@@ -1375,6 +1375,20 @@ overall_header_head_append
 * Since: 3.1.0-a1
 * Purpose: Add asset calls directly before the `</head>` tag
 
+overall_header_headerbar_after
+===
+* Loacations:
```
Comment (marc1706) [inline phpBB/docs/events.md]: Locations

**Comment by marc1706** [comment]
While you're changing the typo, please also change the` Since:` to `3.1.11-RC1`

**Comment by marc1706** [comment]
Hm, didn't expect you to change it this fast. In that case 3.1.10-RC1 would be fine. Sorry for the confusion from my side. ;)

**Comment by marc1706** [comment]
Will squash the commits, fix the commit message, and target version myself. Sorry for the confusion ;)

**Comment by marc1706** [comment]
On a side note, please try to follow the commit message format for all your commits in your next PR:
https://wiki.phpbb.com/Git#Commit_Messages
Thanks

Replacing PR with fixed version

**Pull request closed without merging**